.github/workflows/check-main.yml

```
@@ -18,8 +18,10 @@ jobs:
       with:
           toolchain: stable
           override: true
+    - name: No std
```

I would suggest this as a more accurate name

```suggestion
    - name: Check with no default features
```